command/daemon.go

```diff
@@ -170,8 +170,11 @@ func daemonAction(cctx *cli.Context) error {
 	// Create indexer core
 	indexerCore := engine.New(valueStore, engine.WithCache(resultCache))
 
-	indexCounts := counter.NewIndexCounts(dstore)
-	indexCounts.SetTotalAddend(cfg.Indexer.IndexCountTotalAddend)
+	var indexCounts *counter.IndexCounts
+	if cfg.Ingest.IndexCountsEnabled || cfg.Finder.IndexCountsEnabled {
```

Comment:
Is there a reason to be able to configure the option in other places? I think there should be an enable option in one place that controls the creation of the counter.  If the counter is created, then it gets passed to both ingester and find server, otherwise, nill get passed to both.

Reply:
The rationale for separate flags was that the load pattern on each of those flags is very different. One is pure writes to index counts and the other is pure read. 

Considering explicit options exist already in ingest and finder config, it seems reasonable to also have separate config for them.

Reply:
Consider: If write is not needed, then read cannot be done.  If read is not needed, then there is no reason to write.